google/cloud/internal/rest_carrier.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
```

Comment:
nit: for consistency, move this inside the header guard

Reply:
> For consistency with the library?

Yeah

> I think it makes more sense...

You are probably right. The difference is inconsequential, though.

Reply:
Agreed, it's a no-op either way. Taking the path of least resistance and leaving as is.